src/solver/conv_hip_implicit_gemm_fwd_xdlops.cpp

```diff
@@ -156,8 +156,8 @@ bool PerformanceConfigHipImplicitGemmFwdXdlops::CheckIsSupportCKArgs(
 {
     const auto conv_ptrs = DeviceOpPtrs<DataType>::GetInstances();
     auto ptr_idx         = std::find_if(
-        conv_ptrs.begin(), conv_ptrs.end(), [&kernel_id = this->kernel_id](const auto& ptr) {
-            return ptr->GetTypeString() == kernel_id;
+        conv_ptrs.begin(), conv_ptrs.end(), [&kernl_id = this->kernel_id](const auto& ptr) {
```

Comment:
# 🤣

Reply:
And it depends on installed ROCM, for ROCM 5.6 it does not trigger any warnings.

Reply:
good use of `std::find_if`. Please see if you can repilcate this for other solvers, or better yet, consolidate it into some base-class that all such solvers can use. Thanks.